neuro_flow/storage.py

```diff
@@ -111,6 +112,16 @@ class StartedTask:
     when: datetime.datetime
 
 
+@dataclasses.dataclass(frozen=True)
+class RunningTask:
```

Comment:
I tried not to store `RunningTask` to storage at all, but it makes the continuation (executing restarting for the same attempt) much harder, because some tasks may become ready because another one entered the running state.

Reply:
good point.
I think, on the restart we should do the following:
1. Make sure that all jobs are finished. Otherwise, the attempt is still running and should be explicitly killed before the finish.
2. Restart all not finished jobs. If there is a dependency like `job_a: running` and this dependency is not finished yet -- it will be restarted. If the dependency is already done -- nothing to do here.

The well-organized YAML should finish all running tasks somehow, otherwise it will be running always and require the explicit kill -- which is a sign of logical error.

Reply:
Dropped `RunningTask`